### PR TITLE
chore(app-719): formatting exported design tokens

### PR DESCRIPTION
**Pull request by xReubeNx**

### CODE REVIEW

File: utils/style-dictionary.js [hunk 1]
```diff
@@ -126,7 +142,7 @@ module.exports = (source, destinationDir) => {
   StyleDictionary.registerTransformGroup({
     name: "css",
     transforms: [
-      // based on https://amzn.github.io/style-dictionary/#/transform_groups?id=css
+      // based on .
```
Comment (hongkiulam) [inline utils/style-dictionary.js:145]: was this an accident? 😂

Comment (xReubeNx) [reply utils/style-dictionary.js:145]: ...Yeah, Apologies 😆

File: package.json [hunk 2]
```diff
@@ -21,5 +21,9 @@
   "devDependencies": {
     "style-dictionary": "latest",
     "token-transformer": "latest"
+  },
+  "dependencies": {
+    "json-to-ts": "^1.7.0",
+    "lodash.merge": "^4.6.2"
```
Comment (hongkiulam) [inline package.json:27]: I guess also remove the merge dep, json-to-ts should also be a devdependency

**Review by hongkiulam — APPROVED** [review]
lgtm!

**Pull request merged**